todolist/admin.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib import admin
+
+from .models import *
+
+
+class TodolistAdmin(admin.ModelAdmin):
+    list_display = ('id', 'title', 'time_create', 'photo', 'is_published')
+    list_display_links = ('id', 'title')
+    search_fields = ('title', 'content')
+    list_editable = ('is_published',)
+    list_filter = ('time_create', 'is_published',)
+
+
+class CategoryAdmin(admin.ModelAdmin):
+    list_display = ('id', 'name')
+    list_display_links = ('id', 'name')
+    search_fields = ('name',)
```

Comment:
I would like to know what is happening here, but it's Django magic

Reply:
it's for admin panel, just for better  looking and  filter